tool/make-native-distribution.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+# This script creates a distribution of TruffleRuby and Sulong
+# compiled to a native image by Substrate VM.
+
+# Do not use jt.rb here to minimze dependencies and avoid compatibility problems
+# between the current commit and the built tag's jt.rb.
+
+set -e
+set -x
+
+# Tag to use for the different repositories
+RELEASE=0.32
```

Comment:
Would it make sense to try to get the latest from the `git tag` list?

Reply:
Since it's triggered manually I think it's OK to update this occurrence manually too. Probably we'll need to change this script occasionally anyway when running it.
Also, `git tag` just seems to output in alphabetic order which might not be correct.

Reply:
Newer versions of git have a way to sort the tags. The following appears to work:

```
git tag --sort=committerdate | tail -n 1 | sed -e "s/vm-enterprise-//"
```

But, if you think updating manually is okay, that's fine by me too.